pittapi/laundry.py

```diff
@@ -62,19 +59,89 @@ def get_status_simple(building_name: str) -> Dict[str, str]:
         -> SUTH_EAST
         -> SUTH_WEST
     """
-    laundry_soup = _get_laundry_info(building_name)
-    re_format = re.compile(r"^([0-9]+) of ([0-9]+) available$")
-    washer_text = laundry_soup.find("span", {"id": "washer_available"}).text
-    dryer_text = laundry_soup.find("span", {"id": "dryer_available"}).text
-    washer_match = re_format.match(washer_text)
-    dryer_match = re_format.match(dryer_text)
+    laundry_soup = _get_laundry_soup(building_name)
+    total_washers = 0
+    total_dryers = 0
+    free_washers = 0
+    free_dryers = 0
+
+    machine_dict = parse_soup(laundry_soup)
+    # each building has a different system for washer vs dryer
+    if(building_name == 'SUTH_EAST'):
+        for machine in machine_dict.keys():
+            num = int(machine)
+            if num >= 5 and num <= 14:
```

Comment:
What was the methodology to figure these indices out? out of curiosity.

Reply:
For towers (and I think Sutherland?) there was actually a system with like odds and evens being dryers and washers, but for the smaller buildings it was kind of just however they wanted to arrange it, so I had to organize it manually by going through the site and reading off which ids matched to a washer vs. dryer. That code is me just organizing those ids into their right categories. Not great, but I couldn't figure any other way to do it.